frontend/app/router.js

```diff
@@ -9,4 +9,8 @@ Router.map(function() {
   this.route("search", { path: "/search/:query" });
 });
 
+Router.map(function() {
+  this.resource('legislator', { path: '/legislator/:legislator_id' });
```

Comment:
You can put this inside the other `map` function.  Just cleaner to have one.  Not blocking either